coding-concepts.js

```diff
@@ -5,69 +5,95 @@
 // First, predict the outcome based on your understanding of the code.
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
-// --------------------1) What will this log?
+// // --------------------1) What will this log?
 
-var cohort = "Bravo 2022"
+// var cohort = "Bravo 2022"
 // console.log(cohort.split(""))
 
-// a) Your answer:
+// a) Your answer: I believe once we run the split through the string of "Bravo 2022", we will get an array of substrings. This action will break up our array and cause them to each identified as a string. For example: "B", "r", "a" etc...
 // b) Verify and explain:
 
+// 'B', 'r', 'a', 'v',
+//   'o', ' ', '2', '0',
+//   '2', '2'
+
+// The above was my return and as predicted it broke down my string in to substrings by splitting each value indiviually. 
```

Comment:
spit() will actually turn a string into an array and break it up depending on what is passed in the parenthesis.